vscode-pets-integration/experiment1/scripts/sprite-generation/assemble_gifs.py

```
@@ -0,0 +1,153 @@
+    """Create GIF with clean transparency"""
+    print(f"\n🎬 Creating {anim_name} animation...")
+    print(f"   Frames: {frame_count}")
+    
+    frame_files = []
+    for i in range(1, frame_count + 1):
+        frame_path = os.path.join(FRAMES_DIR, f"{anim_name}_frame_{i:02d}.png")
+        if os.path.exists(frame_path):
+            frame_files.append(frame_path)
+    
+    if not frame_files:
+        print(f"   ❌ No frames found")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)